config/routes.rb

```diff
@@ -12,9 +12,14 @@
   namespace :admins do
     resources :users, only: %i[index show]
     resources :calculators, only: %i[index show new create edit update]
+    resources :admins do
+      get :update_password
```

Comment:
don't use different methods for the same action. `get` and `patch` have absolutely different purpose

Reply:
Fixed this by using `post` with `:update_passwords` and deleted `patch`